features/support/hooks.rb

```diff
@@ -0,0 +1,3 @@
+Before('@login') do
+  click(:id, 'loggout') if $driver.current_url == (ENV['URL'] + '/books')
+end
```

Comment:
Now you have a hooks.rb file, it's a good practice to add a hook to perform some useful actions on the browser after the execution (e.g. clearing local storage or deleting cookies)